src/amo/components/SearchForm.scss

```diff
@@ -10,6 +10,6 @@
 .SearchInput-input,
 .SearchForm-form {
   color: $base-color;
-  font-size: $font-size-m;
+  font-size: 14px;
```

Comment:
We do. Unfortunately from the mocks that search text is like the one place there isn't a shared style and the one place we use `14px` 😕

I'll make this a new size that's just `$font-size-search` because even though it's a one-off I like that the hard-coded pixel value stuck out to you so I'll keep us using vars. 👍